test/build-combination/Common/main.c

```diff
@@ -123,9 +123,12 @@ int main( void )
     return 0;
 }
 /*-----------------------------------------------------------*/
-
-void vApplicationIPNetworkEventHook_Multi( eIPCallbackEvent_t eNetworkEvent,
-                                           struct xNetworkEndPoint * pxEndPoint )
+#if defined( ipconfigIPv4_BACKWARD_COMPATIBLE ) && ( ipconfigIPv4_BACKWARD_COMPATIBLE == 1 )
```

Comment:
Please change:
~~~diff
-    #if defined( ipconfigIPv4_BACKWARD_COMPATIBLE ) && ( ipconfigIPv4_BACKWARD_COMPATIBLE == 1 )
+    #if ( ipconfigIPv4_BACKWARD_COMPATIBLE == 1 )
~~~
because `ipconfigIPv4_BACKWARD_COMPATIBLE` must be defined in FreeRTOSIPConfigDefaults.h.